apps/examples/testcase/le_tc/compression/Kconfig

```diff
@@ -6,6 +6,7 @@
 menuconfig EXAMPLES_TESTCASE_COMPRESSION
 	bool "Compression TestCase Example"
 	default n
+	select COMPRESSION_TEST_DRV
```

Comment:
This should be under TC_COMPRESS_READ.
I think this is needed only in that case.

Reply:
Changes made